generators/app/templates/demos/demo.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
```

Comment:
I'm open to a better naming convention than 'demo' for the dev server that gets spun up when you run npm run start.  Any ideas?  Just dev?